### ISSUE TITLE
Have Aberdeen Autobuild Frontend into .deb

### ISSUE THREAD

**Issue by AnotherOctopus** [purduerov/X11-Core#11]
It would be super nice if every time a pull was merged in, Aberdeen auto built a package and posted it on the aberdeen site. Then anyone could run it on their linux machine super easily and we would have a running working history to fall back to at the pool

**Comment by mattmolo** [comment]
Here is a good package for Electron: [electron-packager](https://github.com/electron-userland/electron-packager)

Very easy to use and you can have it build for all OS's (I'd recommend doing Windows in addition to Linux).

I have used it before for one of my apps, so if you want to see a quick example, look here: [M3-Messenger:package.json](https://github.com/mattmolo/M3-messenger/blob/master/package.json#L8)
Basically, with that setup, all you would need to do is a `npm install && npm run <package-linux>/<package-win>`. If you want to get fancy, you can probably rename the packaged apps with a version a commit hash, and then add some small code to the electron app that would check if it's the latest built version.
